packages/go/openapi/src/paths/sso.sso-providers.oidc.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2024 Specter Ops, Inc.
+#
+# Licensed under the Apache License, Version 2.0
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+parameters:
+  - $ref: './../parameters/header.prefer.yaml'
+post:
+  operationId: CreateOIDCProvider
+  summary: Create OIDC Provider
+  description: Creates a new OIDC provider for SSO authentication
+  tags:
+    - Auth
+    - Community
+    - Enterprise
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: './../schemas/api.requests.oidc-provider.create.yaml'
```

Comment:
I think normally people put their requests and responses into separate definitions.
What we prefer to do is when a schema is only used once, we should inline it since we have no plans to re-use it.

`oidc-provider.create.yaml` can be deleted, and we can have that request inlined here